### PR TITLE
Add splitkb Aurora Lily58 port

### PR DESCRIPTION
**Pull request by flowerseses**
Adding a vial port for the splitkb Aurora Lily58 keyboard - using the current default layout, and with some extra measures to reduce space

### CODE REVIEW

File: keyboards/splitkb/aurora/lily58/rev1/keymaps/vial/vial.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    "lighting": "none",
+    "matrix": {
+        "cols": 6,
+        "rows": 10
+    },
+    "layouts": {
+        "keymap": [
+            {
+                "name": "splitkb.com default Aurora Lily58"
+            },
```
Comment (lesshonor) [inline keyboards/splitkb/aurora/lily58/rev1/keymaps/vial/vial.json]:
```suggestion
```

File: keyboards/splitkb/aurora/lily58/rev1/keymaps/vial/rules.mk [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+# Vial setup
+VIA_ENABLE = yes
+VIAL_ENABLE = yes
+
+# Saving space for the default build, remove all/any of these when using a controller with a RP2040 controller
+OLED_ENABLE = no
+MOUSEKEY_ENABLE = no
+COMBO_ENABLE = no
+KEY_OVERRIDE_ENABLE = no
```
Comment (lesshonor) [inline keyboards/splitkb/aurora/lily58/rev1/keymaps/vial/rules.mk]: For your consideration.
```suggestion
# Saving space on atmega32u4
ifeq ($(strip $(MCU)), atmega32u4)
    OLED_ENABLE = no
    MOUSEKEY_ENABLE = no
    COMBO_ENABLE = no
    KEY_OVERRIDE_ENABLE = no
endif
```

File: keyboards/splitkb/aurora/lily58/rev1/keymaps/vial/vial.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+            [{"x": 3},"0,3",{"x": 6.5},"5,3"],
+            [{"x": 2,"y": -0.95},"0,2",{"x": 1},"0,4",{"x": 4.5},"5,4",{"x": 1},"5,2"],
+            [{"x": 5,"y": -0.85},"0,5",{"x": 2.5},"5,5"],
+	    [{"x": 1,"y": -0.95},"0,1",{"x": 10.5},"5,1"],
```
Comment (lesshonor) [inline keyboards/splitkb/aurora/lily58/rev1/keymaps/vial/vial.json]:
```suggestion
	        [{"x": 1,"y": -0.95},"0,1",{"x": 10.5},"5,1"],
```

**Review by lesshonor — APPROVED** [review]
You should definitely get rid of the extraneous `"name"` attribute in `vial.json`, but otherwise seems fine.

File: keyboards/splitkb/aurora/lily58/rev1/keymaps/vial/rules.mk (hunk 2 above)
Comment (flowerseses) [reply keyboards/splitkb/aurora/lily58/rev1/keymaps/vial/rules.mk]: I'll mess around with this and make sure it works as expected, and commit the change if so, thanks!

**Pull request merged**